src/styles/blocks/stars.css

```diff
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  margin: 0;
+  padding: 0;
```

Comment:
Your code-style is totally nice. Its quite difficult to add something or to suggest some changes in this case...

https://www.w3schools.com/cssref/css_default_values.php

as <div /> has no `margin` and `padding` by default - it's not really nessecary to reset them.

Reply:
You right, fixed